src/lib.rs

```diff
@@ -3,8 +3,6 @@
 //! Full language defined here: <https://google.github.io/xls/dslx_reference/>
 //! Currently a spooky scary skeleton, but actively being built out.
 //!
-//! At present, the _only_ entry point is `parse_function_signature`, taking in an ast::ParseInput.
```

Comment:
I don't think line 6 should be removed along with line 7... was this purposeful?

Reply:
Sigh. I CANNOT HELP BUT MAKE MISTAKES (done).